packages/terra-menu/src/Menu.jsx

```diff
@@ -53,12 +57,21 @@ const propTypes = {
    * Otherwise, the menu will display without an arrow and right aligned.
    */
   isArrowDisplayed: PropTypes.bool,
+  /**
+   * Allows assigning of root element custom data attribute for easy selecting of document base component.
+   */
+  rootSelector: PropTypes.string,
+  /**
+   * Allows assigning of trigger element custom data attribute for easy selecting of triggering component.
+   */
+  triggerElement: PropTypes.object,
```

Comment:
Why is it optional?

Reply:
I had tried making it required but it gives an error saying the value cannot be null.

Reply:
🤔

Reply:
Prop removed in 15a749c